detection_rules/etc/version.lock.json

```diff
@@ -395,45 +417,90 @@
   },
   "16a52c14-7883-47af-8745-9357803f0d4c": {
     "rule_name": "Component Object Model Hijacking",
-    "sha256": "975fcc9572e8117b283322c180c833044bcd17bf6caf3fb3758f1b06c6c48351",
+    "sha256": "0ee9bdc342cc3d58a99e130d3659a7acda929c3b0d61733635486465999c6e76",
     "type": "eql",
-    "version": 6
+    "version": 7
   },
   "16fac1a1-21ee-4ca6-b720-458e3855d046": {
     "rule_name": "Startup/Logon Script added to Group Policy Object",
-    "sha256": "af1ffd24fd0a26c9e8ea2631e0dc2431a63d5292901f4eabdb74a96e7ce20bc5",
+    "sha256": "743ba4eef59cba89f8746fdb4fae26087f3ac2c969a96e7f1f072ea6618a14b5",
     "type": "query",
-    "version": 3
+    "version": 4
   },
   "1781d055-5c66-4adf-9c59-fc0fa58336a5": {
+    "min_stack_version": "8.3",
+    "previous": {
+      "7.16": {
+        "rule_name": "Unusual Windows Username",
+        "sha256": "15ad86ffb8402c2acabbd69bc91cf276320fbefe605de2f336f02d46936242a4",
+        "type": "machine_learning",
```

Comment:
Interesting. Do you know why we min_stacked ML rules?

Reply:
Looks like the ML team min stacked a couple at 8.3. Since we adjusted supported min_stack back to 7.16, these previous min_stacks would have gone from 7.14->7.16 but since it was forked at 8.3 by ML team, a forked version at the minimum of 7.16 was added